README.md

```diff
@@ -1,2 +1,41 @@
-# software-factory
+
+# Software Factory
 PELUX Software Factory documentation
+
+Software Factory is maintained at https://github.com/Pelagicore/software-factory
+
+Maintainer: Joakim Gross <joakim.gross@pelagicore.com>
+
+
+## Dependencies
+* Sphinx
+* sphinxcontrib-seqdiag
+* sphinxcontrib-blockdiag
+* sphinxcontrib-actdiag
```

Comment:
Oh, and I needed texlive-latex-base (on Debian) for pdflatex which is used for generating PDFs